recipes/pychx/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "pyCHX" %}
+{% set version = "4.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bbd5a69119b2b32371edf5363583f3c92933aa7660360543638d6c6ede130228
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+
+requirements:
+  host:
+    - python >=3.6,!=3.9.5,!=3.8.10
```

These restrictions were needed for the `defaults` channel to work in the nsls2forge feedstock. We don't need it for conda-forge.
```suggestion
    - python >=3.6
```